README.md

```diff
@@ -13,6 +13,19 @@ npm install
 npm start
 ```
 
+## Tasks
+
+- `npm start` Run web server.
+- `npm test` Run automated tests.
+- `npm build` Build project.
+- `npm run watch-css` Build LESS files when they change.
+- `npm run watch-js` Build JS files when they change.
+- `npm run watch` Build JS and LESS files when they change.
```

Comment:
You might need to rebase now since we don't have `watch` anymore but we are using `start` for this now.